docs/internals/contributing/writing-code/coding-style.txt

```diff
@@ -159,7 +159,7 @@ Imports
   .. console::
 
       $ python -m pip install "isort >= 5.1.0"
-      $ isort -rc .
+      $ isort -c .
 
   This runs ``isort`` recursively from your current directory, modifying any
   files that don't conform to the guidelines. If you need to have imports out
```

Comment:
I don't think this is correct. From the version [5.0.0 release notes](https://github.com/PyCQA/isort/blob/main/CHANGELOG.md#500-penny---july-4-2020):

> --recursive option has been removed. Directories passed in are now automatically sorted recursive[ly].
--apply option has been removed as it is the default behaviour.

The above code actually checks imports, and reports on files that don't conform to the guidelines but _doesn't_ modify them. To recursively modify all you need is:
```
isort .
```

Reply:
True, it was an issue since its introduction in cd155c7c7160f5178833593713727f013d3ed36d. Fixed.